packages/transmute-framework/src/stream-model/__tests__/sync.spec.js

```diff
@@ -9,6 +9,26 @@ const mockEvents = require('../../__mock__/events.json');
 describe('sync', () => {
   let accounts;
   let eventStore;
+  let streamModel;
+
+  const filter = event => {
+    return event;
```

Comment:
Are you sure filter returns an event object?
It returns both `filter` and `custom_filter` return a boolean in `applyEvents.spec.js`

Reply:
this is me being lazy with javascript type inference. it should return a boolean.

Reply:
Oh I see, got it.